libs/explored-types/src/types.ts

```diff
@@ -132,15 +158,17 @@ export type SearchResultType =
   | 'siacoinElement'
   | 'siafundElement'
   | 'transaction'
+  | 'v2contract'
```

Comment:
Looks like we can update this now: https://github.com/SiaFoundation/explored/pull/197